backend/api/pipelining/_tasks/run.py

```diff
@@ -94,10 +103,12 @@ def run_node_task(run_id: int, node_id: int, previous_job_id: int = None):
         container.remove()
 
 
+# for testing purposes, revert back to 3 in production
 @dramatiq.actor(max_retries=3)
```

Comment:
You can make this a configuration setting 

```suggestion
@dramatiq.actor(max_retries=3)
```